src/components/Book/Books.js

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
+import { createStore } from 'redux';
 import { generate } from 'randomized-string';
 import { alphanumeric } from 'randomized-string/lib/types';
 import Book from './Book';
 import AddBooks from '../Addbooks/AddBooks';
 import './Books.css';
+import reducer from '../../redux/configureStore';
 
 const Books = () => {
-  const books1 = [];
+  const store = createStore(reducer); // creating store from the root reducer
+  console.log('STORE IS', store);
```

Comment:
- [OPTIONAL] Please kindly remove logging statements from your code base as this makes your app vulnerable to attacks.